utils/README.md

```diff
@@ -0,0 +1,134 @@
+# Utilitários do Censo Querido Diário
+
+Este sub-repositório inclui rotinas e funções elaboradas pela comunidade para
+processar, analisar e salvar os resultados do Censo Querido Diário.
+
+Atualmente, o único utilitário desenvolvido é o pacote `fetch_portals`, que se
+comunica com todos os endereços web cadastrados no Censo para checar quais
+portais estão *online* e/ou para obter seu código-fonte.
+
+Contribuições na forma de novos pacotes e utilitários para processar os dados
+do Censo são bem-vindas. Cheque o [CONTRIBUTING.md](../CONTRIBUTING.md) do
+projeto para mais detalhes de como ajudar nas diferentes tarefas do Censo, bem
+como a seção [Adicionando um novo utilitário](#adicionando-um-novo-utilitário)
+para instruções específicas de como criar uma nova rotina de pré-processamento.
+
+Se tiver alguma dúvida ou quiser ter uma visão geral dos próximos passos do
+Censo, não hesite em visitar as
+[issues](https://github.com/okfn-brasil/censo-querido-diario/issues) do projeto
+ou entrar em contato pelo [Discord](https://discord.gg/M6ep5VED).
+
+## Instalação e execução
+
+### Com o Docker (recomendado)
+
+A forma mais simples de rodar os utilitários é utilizando o Docker. Você deve
+ter instalado o Docker Community Edition. Encontre a versão apropriada para o
+seu sistema [aqui][Docker CE]. Você também deve ter o [git] instalado para fazer download do repositório.
+
+Para instalar os utilitários, abra um terminal de linha de comando e rode os
+seguintes comandos:
+
+```bash
+$ git clone https://github.com/okfn-brasil/censo-querido-diario.git
+$ cd censo-querido-diario/utils
+```
+
+Em um explorador de arquivos, encontre o diretório onde você fez download do
+repositório e abra o arquivo `censo-querido-diario/utils/.env.template`. Adapte
+as configurações presentes no arquivo de acordo com os dados que pretende obter
+(especialmente as iniciadas em `KAGGLE_*`, se for exportar para o Kaggle).
+Salve o arquivo modificado renomeie-o para `.env` (sem o `.template` no final).
+
+Para inicializar a checagem dos portais, basta voltar ao terminal e inserir o
+comando:
+
+```bash
+$ docker-compose up
+```
+
+[Docker CE]: https://hub.docker.com/search?offering=community&type=edition
+[git]: https://git-scm.com/
+
+### Como pacote Python
+
+Os utilitários contidos nesse sub-repositório podem ser instalados como pacotes
+Python avulsos. Para isso, você deve ter instalada na sua máquina uma versão
+Python compatível (3.7 ou superior).
```

Comment:
I see you use `typing.Final` in the code, but [`typing.Final` was introduced in Python 3.8](https://docs.python.org/3/library/typing.html#typing.Final). There's also a Python 3.7y reference in `setup.cfg`.